skrifa/generated/autohint_scripts.py

```diff
@@ -0,0 +1,1385 @@
+# Generates Rust tables that define Unicode "script classes" for the purposes
```

Comment:
I find it confusing that this lives in the `generated` folder; I'd generally expect that to only contain generated files (e.g. it should be safe to blow it away when rebuilding).

Is this hand-written?

Reply:
This is a combined of stuff that was very painfully scraped from FreeType along with some hand-written code. Itis intended to be our source of truth forevermore so should hopefully not be blown away.

Moved to `skrifa/scripts/gen_autohint_scripts.py`.